model-test/pom.xml

```diff
@@ -58,6 +58,21 @@
 			<artifactId>guice</artifactId>
 			<version>4.2.3</version><!--$NO-MVN-MAN-VER$-->
 		</dependency>
+
+		<dependency>
+			<groupId>hu.blackbelt.judo</groupId>
+			<artifactId>judo-requirement-report-annotation</artifactId>
+			<version>1.0.0.20230105_022024_241fcd45_feature_JNG_4297_Creating_Judo_test_methodology</version>
+			<scope>compile</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>hu.blackbelt.judo</groupId>
+			<artifactId>judo-requirement-report-processor</artifactId>
+			<version>1.0.0.20230105_022024_241fcd45_feature_JNG_4297_Creating_Judo_test_methodology</version>
+			<optional>true</optional>
+			<scope>compile</scope>
+		</dependency>
```

Comment:
Do not use direct version reference in dependency for our nodule.

1. Please create a property in parent pom.xml in block 1.0.0.20230105_022024_241fcd45_feature_JNG_4297_Creating_Judo_test_methodolog
2. Put this dependency to parent's pom section with version using the property defined earlier ${judo-requirement-report-version}
3. In this place use the dependency without version and scope.
4. When a new property is defined in a parent's pom referencing our module, in judo-ng's (and judo-community too in this case) project-meta.yaml have to add the module as dependency. In our case in the judo-runtime-core JSL entry will have a new dependency which is judo-requirement-report.

Reply:
@robertcsakany
The last sentence correctly this: "In our case in the judo-meta-jsl JSL entry will have a new dependency which is judo-requirement-report.", is it?